exe/cli/cmd/helpers.go

```diff
@@ -0,0 +1,117 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/logrusorgru/aurora"
+	"github.com/mitchellh/go-homedir"
+	ma "github.com/multiformats/go-multiaddr"
+	"github.com/olekukonko/tablewriter"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+type Flag struct {
+	Key      string
+	DefValue interface{}
+}
+
+func InitConfig(v *viper.Viper, file string, defDir string, name string) func() {
+	return func() {
+		if file != "" {
+			v.SetConfigFile(file)
+		} else {
+			home, err := homedir.Dir()
+			if err != nil {
+				panic(err)
+			}
+			v.AddConfigPath(path.Join("./", defDir)) // local config takes priority
+			v.AddConfigPath(path.Join(home, defDir))
+			v.SetConfigName(name)
+		}
+
+		v.SetEnvPrefix("TXTL")
+		v.SetEnvKeyReplacer(strings.NewReplacer(".", "_"))
+		v.AutomaticEnv()
+		_ = v.ReadInConfig()
+	}
+}
+
+func BindFlags(v *viper.Viper, root *cobra.Command, flags map[string]Flag) error {
```

Comment:
I don't see this being used... should we wire it up?

Reply:
Copy pasta. Not sure if I want to wire it up. I've been explicit about what flags are defined where and bind viper as needed. Please correct me if I'm wrong, but my reasoning is that using something like this `BindFlags` tends to cause you to overload the root command with persistent flags that don't actually all apply to every child command. My goal was to keep the auto generated command help accurate in the sense that global persistent flags actually apply to every child and that child flags accurately represent the flags available for that command.

Reply:
I'll delete the things in this file that aren't being used.